### DISCUSSION TITLE
Confused about Project Structure and Makefiles

### DISCUSSION THREAD

**Discussion by Abgeleitet** [omnetpp/omnetpp#1172] [Q&A]
So I have the following project structure:

lin
Makefile
│   ├── simulations
│   │   ├── Lin.ned
│   │   ├── omnetpp.ini
│   │   ├── package.ned
│   ├── src
│   │   ├── LinMaster.cc
│   │   ├── LinMaster.ned
│   │   ├── LinSlave.cc
│   │   ├── LinSlave.ned
│   │   ├── package.ned

I am confused where to have my Makefile and where to run my Project.
Using opp_makemake i created a Makefile in the root directory of my Project. 
After some research i found out i can create a "deep" Makefile using  opp_makemake --deep.
But i still get the following error when trying to run ./out/clang-release/lin : 
`Error cannot open ini file 'omnetpp.ini`

which makes sense to me because its not in the current directory. 
What is the correct way to run this project with this kind of project structure.
Thanks in Regards.

**Comment by rhornig** [comment]
If you keep the simulation and src folder seprated, then the usual way is to have a hand written makefile in the top-level dir and an opp_makemake generated one in the src folder.

Your best bet is to create a new project with simulation/src folder using the IDE (with new project wizard) which will set up all necessary files as needed.
